.github/workflows/announce-lts-rc.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           discourse-author-username: jenkins-release-bot
           discourse-category: 23
       - name: Post on mailing list
-        uses: dawidd6/action-send-mail@v3
+        uses: dawidd6/action-send-mail@v4
```

Comment:
Should use hash instead.

Reply:
@daniel-beck For what reason?

Reply:
General good practice, see e.g. https://github.com/ossf/scorecard/blob/51f31c9882b6e5998e0df571096147a99842092b/docs/checks.md#pinned-dependencies for OpenSSF's take on this.

Previously done elsewhere in https://github.com/jenkinsci/jenkins/pull/7584.

Reply:
Is that not a pre-existing issue? How is this PR introducing a problem?